demos/fonts/all.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <div id="outputBravura"></div>
+    <div id="outputGonville"></div>
+    <div id="outputPetaluma"></div>
+    <!-- The standard vexflow build includes three music fonts. -->
+    <script src="../../build/vexflow.js"></script>
+    <script type="module">
+      console.log('VexFlow BUILD ' + Vex.Flow.BUILD);
+
+      ////////////////////////////////////////////////////////////////////////
+
+      {
+        Vex.Flow.setMusicFont('Bravura');
+        const factory = new Vex.Flow.Factory({
+          renderer: { elementId: 'outputBravura', width: 500, height: 130 },
+        });
+        const score = factory.EasyScore();
+        factory
+          .System()
+          .addStave({
+            voices: [
+              score.voice(score.notes('C#5/q, B4, A4, G#4', { stem: 'up' })),
+              score.voice(score.notes('C#4/h, C#4', { stem: 'down' })),
+            ],
+          })
+          .addClef('treble')
+          .addTimeSignature('4/4');
+        factory.draw();
+      }
+
+      ////////////////////////////////////////////////////////////////////////
+
+      {
+        Vex.Flow.setMusicFont('Gonville');
```

Comment:
You can change the global music engraving font. The cleanest way is to instantiate a new Factory for each font you want to use on the same webpage. If you change fonts between different sets of voices and staves, but use the same Factory, it won't do what you expect. That's because a single Factory has a loop for drawing all of its internal members in a particular order. 

Anyways, to simplify things, in the Font PR I removed the ability for a single Element to have its own music font. We always use the global music font stack now.

Reply:
Your first paragraph is also useful as an embedded comment.

As discussed, +1 for removing per-Element fonts.